content/docs/csidriver/upgradation/drivers/isilon.md

```diff
@@ -28,5 +27,5 @@ You can upgrade the CSI Driver for Dell PowerScale using Helm or Dell CSI Operat
    Edit the _my-isilon-settings.yaml_ as per the requirements.
 
       ```bash
-      ./csi-install.sh --namespace isilon --values ./my-isilon-settings.yaml --upgrade
+      ./csi-install.sh --namespace isilon --values ./my-isilon-settings.yaml  --helm-charts-version csi-isilon-2.8.0 --upgrade
```

Comment:
Is it mandatory to mention `helm-chart-version`?

Reply:
to upgrade particular version we need helm-chart-version